core/src/main/java/apoc/nodes/Nodes.java

```diff
@@ -653,20 +653,20 @@ public boolean isDense(@Name("node") Node node) {
     }
 
     @UserFunction("apoc.any.isDeleted")
-    @Description("returns boolean value for nodes and rele existance")
-    public boolean isDeleted(@Name("thing") Object thing) {
-        if (thing == null) return true;
+    @Description("Returns true if the given node or relationship no longer exists.")
```

Comment:
This is just my copy for it, I'll get Jens to double check it (the original is so bad I couldn't leave it 🙈)

Reply:
So this one was not documented at all?